src/ekore/operator_matrix_elements/unpolarized/time_like/as1.py

```diff
@@ -0,0 +1,98 @@
+"""The unpolarized, time-like |NLO| matching conditions."""
+
+import numba as nb
+import numpy as np
+
+from eko.constants import CF
+
+
+@nb.njit(cache=True)
+def A_hg(N, L):
+    r"""Compute the |NLO| heavy-gluon |OME|.
+
+    Implements :eqref:`27` from :cite:`Cacciari:2005ry`.
+
+    Parameters
+    ----------
+    N : complex
+        Mellin moment
+    L : float
+        :math:`\ln(\mu_F^2 / m_h^2)`
+
+    Returns
+    -------
+    A_hg : complex
+        |NLO| heavy-gluon |OME|
+        :math:`A_{hg}^{S,(1)}`
+
+    """
+    result = (
+        2
+        * CF
+        * (
+            (2 + N + N**2) / (N * (N**2 - 1)) * (L - 1)
+            + 4 / (N - 1) ** 2
+            - 4 / N**2
+            + 2 / (N + 1) ** 2
+        )
+    )
+    return result
+
+
+@nb.njit(cache=True)
+def A_gg(L):
+    r"""Compute the |NLO| gluon-gluon |OME|.
+
+    They are defined as the Mellin transform of :eqref:`B.6` from :cite:`Buza_1998`.
```

Comment:
If you copy, please
1. put the Cacciari equation as reference and say it is the same as space-like
2. add a unit test to check they are the same (it is trivial, I know)